mule.yaml

```diff
@@ -79,29 +39,19 @@ tasks:
 
   - task: s3.BucketCopy
     name: setup-deb
```

Comment:
Missed this before, should the name be `upload-deb`?

Reply:
This was intended to download the package from s3 if someone had previously built and pushed the packages.  The idea was to be able to have devs run tasks "a la carte" as another option to just running one task after another.  I don't know if this applies to the indexer project.

The name isn't the greatest.